cmd/kubeadm/app/preflight/checks_other.go

```diff
@@ -0,0 +1,55 @@
+//go:build !linux
+// +build !linux
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package preflight
+
+import (
+	system "k8s.io/system-validators/validators"
+	utilsexec "k8s.io/utils/exec"
+)
+
+// addOSValidator add os validators
+// No-op for Darwin (MacOS), Windows.
```

Comment:
please see my suggestions from the _linux file. copy them here but also add the lines:
```
// functionName ... text from _linux file
// No-op for Darwin (MacOS), Windows.
```